src/components/Navigation.js

```diff
@@ -1,9 +1,63 @@
-export default function Navigation(props) {
+import { useCallback, useEffect } from "react";
+
+function Navigation ({ data, currentIndex, setCurrentIndex, popoverOpen }) {
+    const handleNextData = useCallback(() => {
+        const length = data.length;
+        var index = currentIndex + 1;
+        index = index % length;
+        setCurrentIndex(index);
+    }, [setCurrentIndex, currentIndex, data]);
+    
+    const handlePreviousData = useCallback(() => {
+        const length = data.length;
+        var index = currentIndex;
+        index = index === 0 ? length - 1 : index - 1;
+        setCurrentIndex(index);
+    }, [setCurrentIndex, currentIndex, data]);
+
+    const handleNextUnverified = useCallback(() => {
+        let index = data.findIndex( (currentData, idx) => idx > currentIndex && !currentData.status);
+        index = index === -1 ? data.findIndex( (currentData) => !currentData.status) : index;
+        setCurrentIndex(index);
+    }, [setCurrentIndex, currentIndex, data]);
+
+    const handlePrevUnverified = useCallback(() => {
+        let index = data.findLastIndex( (currentData, idx) => idx < currentIndex && !currentData.status);
+        index = index === -1 ? data.findLastIndex( (currentData) => !currentData.status) : index;
+        setCurrentIndex(index);
+    }, [setCurrentIndex, currentIndex, data]);
```

Comment:
> I was able to get a crash to occur.

Aaaah, that's embarrassing! I have changed the implementation in [a48e5de](https://github.com/Automattic/compare-snapshot-tool/pull/7/commits/a48e5debe0910fdb3dbfb794745b6442710c8892). 

I have added another check for the index in case every item in CSV has data. The current index will stay the same and an alert will be displayed that there are no unverified items left.